### PR TITLE
entity-groups-refactor

### PR DESCRIPTION
**Pull request by kghamilton89**

### CODE REVIEW

File: configuration/entity_groups.md [hunk 1]
```diff
@@ -2,25 +2,25 @@
...
 
-It can be also created by specifying a boolean criteria against which all entities that exist in the database, or parent group entities, are evaluated. Those entities that match the criteria are added as members **automatically**. Such groups are continuously updated in the background.
+It can be also created **programmatically** by specifying a boolean criteria against which all entities that exist in the database, or parent group entities, are evaluated. Those entities that match the criteria are added as members **automatically**. Such groups are continuously updated in the background.
 
 The parent group, if specified, restricts possible members to members of the parent group.
 
-The entity group editor provides the following options for adding or removing members:
+The Entity Group Editor provides the following options for adding or removing members:
 
 ### Text List
```
[the remark this answers is not in the document]
Comment (kghamilton89) [reply configuration/entity_groups.md]: Screenshots added.

File: configuration/entity_groups.md [hunk 2]
```diff
@@ -2,25 +2,25 @@
 
 ## Overview
 
-Entity Groups organize similar entities into logical collections that can be re-used when managing user permissions, filtering data, or calculating aggregations.
+Entity Groups organize similar entities into logical collections that can be used when managing user permissions, filtering data, or calculating aggregations.
 
 ![](./images/entity-groups.png)
```
[the remark this answers is not in the document]
Comment (kghamilton89) [reply configuration/entity_groups.md]: Updated

File: configuration/entity_groups.md [hunk 3]
```diff
@@ -75,57 +75,57 @@ name LIKE '*vml*' && tags.location = 'NUR'
...
   ```javascript
   properties('cfg').prog != ''
     && properties('cfg').prog NOT LIKE 'topas*'
   ```
 
-* Entity is a member of another group
+* Entity is a member of another group:
 
   ```javascript
   memberOf('all-linux-servers')
```
[the remark this answers is not in the document]
Comment (kghamilton89) [reply configuration/entity_groups.md]: added

**Comment by kghamilton89** [comment]
* Requested changes made, screenshots added, Entity Views section added.
* Refactored `entity_views.md` because I read it to add the EV section to the Groups document. Most of the changes were trivial (capitalizing Entity Views throughout the document for example), but there were some actual grammatical changes as well.

**Comment by rodionos** [comment]
Please, lets go back to lower case. I can see some utility but It's too much to keep in mind when writing docs.

**Comment by kghamilton89** [comment]
Reverted back to lowercase.

**Pull request merged**